web/src/main/java/se/inera/intyg/webcert/web/csintegration/integration/CSIntegrationService.java

```diff
@@ -187,6 +199,33 @@ public Certificate createCertificate(CreateCertificateRequestDTO request) {
         return response.getCertificate();
     }
 
+    public CreateDraftCertificateResponseType createDraftCertificate(CreateCertificateRequestDTO request, IntygUser user) {
+        final var url = baseUrl + CERTIFICATE_ENDPOINT_URL;
+        try {
+            final var response = restTemplate.postForObject(url, request, CertificateServiceCreateCertificateResponseDTO.class);
+
+            if (response == null) {
+                throw new IllegalStateException(NULL_RESPONSE_EXCEPTION);
+            }
+
+            pdlLogService.logCreatedWithIntygUser(response.getCertificate(), user);
```

Comment:
Why do we perform the pdl and monitorlog here instead of following the pattern and performing it in the CS-service?

Reply:
Valid point, I will correct this error.